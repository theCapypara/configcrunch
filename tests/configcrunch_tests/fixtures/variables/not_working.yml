base:
  level_dict:
    doc1:
      name: "{{ more.name }}"
      more:
        name: doc1
    doc2:
      name: "{{ more.name }}"
      more:
        name: test
        doc1: "{{ parent().level_dict.doc1.name }}"
        doc2: "{{ name }}"
        doc3: "{{ parent().level_dict.doc3.name }}"
        parent_a: "{{ parent().more.a }}"
    doc3:
      name: "{{ more.name }}"
      more:
        name: doc3

  more:
    a: "{{ more.b }}"
    b: b