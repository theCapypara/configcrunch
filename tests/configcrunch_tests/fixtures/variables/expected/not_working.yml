base:
  level_dict:
    doc1:
      name: doc1
      more:
        name: doc1
    doc2:
      name: test
      more:
        name: test
        doc1: doc1
        doc2: test
        doc3: test # "BUG"! See test. -- Because {{ parent().level_dict.doc3.name }} -> {{ more.name }} -> test
        parent_a: "" # "BUG"! See test.
    doc3:
      name: doc3
      more:
        name: doc3

  more:
    a: b
    b: b